README.md

```diff
@@ -457,6 +457,8 @@ using a combination of `bwa` and `samtools` to output a BAM file instead of a SA
 
 - If the tool supports multi-threading then you MUST provide the appropriate parameter using the Nextflow `task` variable e.g. `--threads $task.cpus`.
 
+- If a module contains _two_ tools that supports multi-threading (e.g. [piping output into a samtools command](https://github.com/nf-core/modules/blob/28b023e6f4d0d2745406d9dc6e38006882804e67/modules/bowtie2/align/main.nf#L32-L46)), split `task.cpus` into a new variable that are given to both tools, allowing both to utilise the provided CPUs.
```

Comment:
```suggestion
- If a module contains _multiple_ tools that supports multi-threading (e.g. [piping output into a samtools command](https://github.com/nf-core/modules/blob/28b023e6f4d0d2745406d9dc6e38006882804e67/modules/bowtie2/align/main.nf#L32-L46)), you MUST assign cpus per tool such that the total number of used CPUs does not exceed `task.cpus`. 
```

I didn't get the part about the variable... Also it doesn't necessarily make sense to split cpus 50:50?

Reply:
Good point, (once again) PR was in response to one issue on one module PR.

By the variable I'm referring specifically to the [`split_cpus` variable](https://github.com/nf-core/modules/blob/28b023e6f4d0d2745406d9dc6e38006882804e67/modules/bowtie2/align/main.nf#L32)  in the example. I 'm assuming it makes sense to have that consistent across all modules.